package.json

```diff
@@ -23,7 +27,11 @@
     "eslint-plugin-jsx-a11y": "6.3.1",
     "eslint-plugin-prettier": "3.1.4",
     "eslint-plugin-react": "7.20.6",
-    "prettier": "2.1.2"
+    "gatsby": "^4.24.7",
```

Comment:
what about gatsby 5 compat?

Reply:
Yea we probably just need it as a peer dependency.